src/omero_zarr/masks.py

```diff
@@ -165,20 +189,15 @@ def save(self, masks, name):
             )
 
         # Setting za.attrs[] doesn't work, so go via parent
-        if "0" in root:
-            image_name = "../../0"
+        image_attrs = {}
+        if image_array_url:
+            image_attrs["array"] = image_array_url
+        elif "0" in root:
+            image_attrs["array"] = image_array_url
```

Comment:
Is `image_array_url` not `''` or `None` in that case? Should this be `../../0` as above?

Reply:
You're right, it's a copy-and-paste error